lib/shopify_api/auth.rb

```diff
@@ -0,0 +1,26 @@
+# typed: strict
+# frozen_string_literal: true
+
+module ShopifyAPI
+  module Auth
+    extend T::Sig
+
+    class << self
+      extend T::Sig
+
+      sig { params(host: T.nilable(String)).returns(String) }
+      def embedded_app_url(host)
```

Comment:
I think `ShopifyAPI::Auth.embedded_app_url` is a sane name, wondering if others feel differently.